### ISSUE TITLE
Not handled errors leading to stop responding and memory overallocation

### ISSUE THREAD

**Issue by pawel-id** [vlocityinc/vlocity_build#406]
We have constant issues with stability of VBT since introduction of LWC activation via puppeteer. We are using latest version of VBT 1.14.11 
It is eating a lot of memory with LWC activations, stopping resonding in some cases and  sometimes crashing due to out of memory.

Here is possible source of the issues we (and maybe other have). Lack of handling of timeout when opening puppeteer page and not calling browser.close(). This lead to left chrome instance (memory consumption) and unable to close the VBT after deployment. 

See the following lines of code. Timeout during page.open(...) is not handled by error handler with browser.close()
https://github.com/vlocityinc/vlocity_build/blob/v1.14.11/lib/datapacktypes/omniscript.js#L219
https://github.com/vlocityinc/vlocity_build/blob/v1.14.11/lib/datapacktypes/omniscript.js#L228

**Comment by rutlabaga** [comment]
The latest release should fix the issue of it not closing the browser, but may not fix whatever the underlying issue is. But it will now correctly report the error. Please let me know if it is working better.

**Comment by pawel-id** [comment]
Thank you, I did apply the latest  1.14.12 and can confirm that the VBT crash rate has dropped (it’s non zero though, but improvement is substantial). Now all the failed activations instead of crashing deployment are silently swallowed with the error message (real example:  `LWC Activation Error >> OmniScript/account_changeTechnicalMigrationPlannedDate_English - TimeoutError: Navigation timeout of 30000 ms exceeded`). This is not crashing. The deployment completes successfully regardless of lack of activation of some OmniScripts. Unfortunately this way we moved immediate issues with failing deployment to later (harder to detect) issues with non active LWC OmniScripts.

Having said, we are grateful for prompt change, but we are waiting for further fixes making VBT LWC activations reliable.
